base_layer/wallet_ffi/wallet.h

```diff
@@ -159,6 +159,13 @@ struct TariPrivateKey *private_key_from_hex(const char *hex, int *error_out);
 // Frees memory for a TariPrivateKey
 void private_key_destroy(struct TariPrivateKey *pk);
 
+/// -------------------------------- Mnemonic Word List  ------------------------------------------ ///
+// Create a TariMnemonicWordList instance containing the entire mnemonic wordlist for the requested language
+struct TariMnemonicWordList *mnemonic_word_list_for_language(const char *language, int *error_out);
+
```

Comment:
You'll need a forward declaration at the top of this file:
```
struct TariMnemonicWordList;
```

You'll need two additional functions here with the implementation in rust:
```
unsigned int mnemonic_word_list_get_length(struct TariMnemonicWordList *mnemonic_word_list, int *error_out);

char * mnemonic_word_list_get_at(struct TariMnemonicWordList *mnemonic_word_list, int *error_out);
```

Reply:
There is already a TariSeedWords object which is what we should be using for this.

Reply:
I think the Word List (set of possible words) is different to TariSeedWords (recovery phrase words) here @philipr-za - but I might be wrong

Reply:
@StriderDM is there an integration test we could add to ensure the header file has all the necessary declarations? checking it manually will eventually result in something getting missed

Reply:
> I think the Word List (set of possible words) is different to TariSeedWords (recovery phrase words) here @philipr-za - but I might be wrong

I agree and if this wasn't an FFI interface I would prefer a distinctly named type but in FFI if you forward declare a new object now you also have to provide all the helper functions for a client to interact with it...when we could just reuse the TariSeedWords object which is the exact same thing with maybe a debatable not quite appropriate name.

Reply:
Oh I see your point, makes sense